Piwik.Tracker/PiwikTracker.cs

```diff
@@ -1280,6 +1279,7 @@ protected bool LoadVisitorIdCookie()
             {
                 _visitCount = long.Parse(parts[2]);
             }
+            // todo _currentVisitTs is never used (was earlier assigned with null in constructor)

```

Comment:
This is the same behavior as in the PHP lib :

 - https://github.com/piwik/piwik-php-tracker/blob/master/PiwikTracker.php#L149
 - https://github.com/piwik/piwik-php-tracker/blob/master/PiwikTracker.php#L1285

However, in PHP, instance variables are public by default. This means the content of this field is made available to users of the library.

Is it bad practice if we keep the code as is?